File: ItineraryBuilder-Backend/Models/Photo.cs
using System.Text.Json.Serialization;

namespace ItineraryBuilder_Backend.Models
{
    public class Photo
    {
        public int Id { get; set; }
        public string Url { get; set; }

        public int PlaceId { get; set; }
        [JsonIgnore]
        public Place Place { get; set; }
       
    }
}

File: itinerarybuilder/lib/models/photo.dart
class Photo {
  final int Id;
  final String Url;

  Photo({
    required this.Id,
    required this.Url
  });

  factory Photo.fromJson(Map<String, dynamic> json) {
    return Photo(
      Id: json['id'],
      Url: json['url'],
    );
  }
}